pinc/charsuite-basic-latin.inc

```diff
@@ -11,15 +11,21 @@ $charsuite->codepoints = [
     // without the soft hyphen
     'U+00a1-U+00ac',
     'U+00ae-U+00ff',
-    // codepoints that map to Windows-1252
-    'U+0152', // OE ligature
+    // codepoints that map to Windows-1252 + extras
+    'U+0128', // [~I] (not in CP-1252)
+    'U+0129', // [~i] (not in CP-1252)
+    'U+0152', // OE ligature~
```

Comment:
Errant `~` at the end of the comment?

Reply:
Fixed, thanks -- no clue how that got in there.